### ISSUE TITLE
Enable remapping the whole behavior of 'esc' key to another key

### ISSUE THREAD

**Issue by f4z3r** [Canop/broot#740]
I like to remap `Esc` to `Ctrl-y`. I thus have two bindings I am mostly interested in rebinding: `:back`, and `:input_clear`. If I define both, only the first in the configuration gets picked up.
```
$ broot --version
broot 1.25.0
```

**Comment by Canop** [comment]
I don't follow what you did.
Can you please paste the relevant part of the configuration (i.e. what you added) and the [debug log](https://dystroy.org/broot/community/#log) ?

**Comment by f4z3r** [comment]
@Canop sorry, turns out it did behave "as expected". It executed `:back` in the while the input was still active. It's a little strange, but I guess it makes sense once I think about it. My issue is that I would like to have something such as:

```toml
[[verbs]]
internal = ":input_clear"
key = "ctrl-y"

[[verbs]]
internal = ":back"
key = "ctrl-y"
```

In my config. That way I can use `Ctrl-y` the same way `Esc` behaves by default. However, if I write this into my config, only the first binding will actually be respected, and the second is ignored.

**Comment by Canop** [comment]
What you're trying to do is to have `ctrl-y` behave like the `esc` key does in standard ?

Right now, the behavior of the esc key is hardcoded, it is one of those, by priority:
1. if there's a "tab cycle" (i.e. a `tab` driven choice between possible completions), it is cancelled
2. if the "modal mode" is enabled and we're in insertion mode, then leave it
3. clear the input and executes `:back`

It's not really possible with the current keybinding system to configure another key to have this complete behavior.

**Comment by Canop** [comment]
Part of what you want could probably be done with

```
    {
        key: ctrl-y
        cmd: ":input_clear;:back"
    }
```

**Comment by f4z3r** [comment]
Yes, perfect that works! Thank you very much. Didn't think of using `cmd` for it.

**Comment by Canop** [comment]
Unfortunately this doesn't make it easy to end a completion cycle :\

**Comment by f4z3r** [comment]
Ah, for me the current workaround is enough. I feel like I don't really cancel completions very often, and can then still resort to `esc` or deleting the completed part 👍🏽

**Comment by Canop** [comment]
I modified the title and category because what I get for this is that it would be good to have a way to replace the 'esc' key with another one.

**Comment by Canop** [comment]
The `:escape` internal is available with broot 1.26.0
